.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.wasm
-**/proptest-regressions
\ No newline at end of file
+**/proptest-regressions
+/tests/contracts/*.wat
```

Comment:
Where does that come from?

Reply:
That's an artifact of the way I test sometimes -- manually generating the wasm, then converting to wat to either manually review, or to see the errors reported by `wat2wasm`. Regardless, I don't think there is any reason that we would want to commit .wat files here, so should be safe to ignore.